app/src/main/res/layout/change_password.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".ui.user.ChangePasswordFragment">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/change_password"
+        android:textFontWeight="600"
+        android:textColor="@color/black"
+        android:textSize="20dp"
+        android:textStyle="normal"
+        android:layout_marginTop="50dp"
+        android:layout_marginBottom="20dp"
+        android:layout_gravity="center_horizontal"></TextView>
+
+    <TextView
+        android:id="@+id/text_old_password"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:clickable="true"
+        android:fontFamily="@font/font_poppins"
+        android:text="@string/old_password"
+        android:textColor="@color/black"
+        android:textSize="19dp"
+        android:textStyle="normal"
+        tools:ignore="TouchTargetSizeCheck"></TextView>
```

Comment:
You can remove the empty tag </TextView> and this can be applied for all empty tags in this layout. Which IDE are you using to write code? If you are using Android studio there are some helpful code inspection warnings and errors: https://www.jetbrains.com/help/idea/inspections-settings.html

Reply:
addressed